### PR TITLE
Add release with vSphere CAPV controllers

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: helm/releases-vmware/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```
Comment (MarcelMue) [inline helm/releases-vmware/Chart.yaml:1]: I think you will need more changes to actually make this "deployable"

Comment (MarcelMue) [reply helm/releases-vmware/Chart.yaml:1]: See also here:
https://github.com/giantswarm/releases/blob/760c7d726591e84ded633847c7d6907d0cfbe077/.circleci/config.yml#L29

Comment (QuentinBisson) [reply helm/releases-vmware/Chart.yaml:1]: it is highly likely but I cannot really test is anyway :D

Comment (QuentinBisson) [reply helm/releases-vmware/Chart.yaml:1]: It should be fixed now :D

**Comment by tfussell** [comment]
Closing in favor of https://github.com/giantswarm/releases/pull/804

**Pull request closed without merging**